test/Room-test.js

```diff
@@ -0,0 +1,52 @@
+import { expect } from "chai";
```

Comment:
Nice, thorough testing. Doesn't seem like there would be much of a sad path for Room, so this looks good. For other classes just don't forget to include sad paths and stuff.